### PR TITLE
Updated `CONTRIBUTING.md` to direct contributors to the pf2e foundry discord server

### PR DESCRIPTION
**Pull request by Fin-bar**

### CODE REVIEW

**Comment by stwlam** [comment]
1. Please update the PR title to reflect what you're doing with more specificity than updating a file
2. Just link to the pf2e server instead if you're wanting to add it.

**Comment by Fin-bar** [comment]
Could you elaborate on bullet point 2? I *did* include a link to the server. Do you mean *only* include a link to the pf2e discord as opposed to the general foundry one?

**Comment by Fin-bar** [comment]
Under the assumption that my last comment is what you're looking for, I pushed a new commit to avoid waiting on asynchronous communication.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Welcome All
 
-If you would like to contribute to the project then I welcome all support. If you are a developer (or would like to be) the hit us up in discord (#pf2e in https://discord.gg/foundryvtt) and we can help get you up and running. Merge requests are welcome and there are plenty of issues logged if you want to sink your teeth into something.
+If you would like to contribute to the project then I welcome all support. If you are a developer (or would like to be) the hit us up on discord (https://discord.gg/pf2e) and we can help get you up and running. Merge requests are welcome and there are plenty of issues logged if you want to sink your teeth into something.
```
Comment (stwlam) [inline CONTRIBUTING.md:5]:
```suggestion
If you would like to contribute to the project then I welcome all support. If you are (or would like to be) a developer, then hit us up on discord (https://discord.gg/pf2e) and we can help get you up and running. Merge requests are welcome, and there are plenty of issues logged if you want to sink your teeth into something.
```

**Comment by Fin-bar** [comment]
Sorry for force push; I didn't like the automated commit message from accepting the suggestion

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Welcome All
 
-If you would like to contribute to the project then I welcome all support. If you are a developer (or would like to be) the hit us up in discord (#pf2e in https://discord.gg/foundryvtt) and we can help get you up and running. Merge requests are welcome and there are plenty of issues logged if you want to sink your teeth into something.
+If you would like to contribute to the project then I welcome all support. If you are (or would like to be) a developer, then hit us up on discord (https://discord.gg/pf2e) and we can help get you up and running. Pull requests are welcome, and there are plenty of issues logged if you want to sink your teeth into something.
```
Comment (stwlam) [inline CONTRIBUTING.md:5]:
```suggestion
If you would like to contribute to the project then I welcome all support. If you are (or would like to be) a developer, then hit us up on Discord (https://discord.gg/pf2e) and we can help get you up and running. Pull requests are welcome, and there are plenty of issues logged if you want to sink your teeth into something.
```

**Pull request merged**